build.gradle

```diff
@@ -74,7 +75,13 @@ task( compileTestTaxonomiesInflection, dependsOn: compileTestModelJava, type: Ja
 	main = "ch.liquidmind.inflection.compiler.InflectionCompiler"
 	classpath = sourceSets.main.runtimeClasspath + sourceSets.testModel.runtimeClasspath
 	jvmArgs '-ea'
-	args "${buildDir}/resources/testModel", "BOOTSTRAP", "src/test/resources/ch/liquidmind/inflection/test/model/TestModel.inflect"
+	args "${buildDir}/resources/testModel", "BOOTSTRAP", 
```

Comment:
I think we should consider running the compiler(s) and proxy generator from within the tests themselves. The workflow in Gradle makes sense for applications that *use* Inflection, but for our own tests I think it might be better for any errors to show up in the junit results. To make this work we would need to use a custom class loader; we'll also have to compile the Java classes using the Java tools API (analogous to Deflector).